test/k8s-integration/main.go

```diff
@@ -678,14 +673,33 @@ func runTestsWithConfig(testParams *testParameters, testConfigArg, reportPrefix
 		kubeTest2Args = append(kubeTest2Args, fmt.Sprintf("--artifacts=%s", kubetestDumpDir))
 	}
 	kubeTest2Args = append(kubeTest2Args, "--")
-	if len(*testVersion) != 0 && *testVersion != "master" {
-		kubeTest2Args = append(kubeTest2Args, fmt.Sprintf("--test-package-marker=latest-%s.txt", *testVersion))
+	if len(*testVersion) != 0 {
+		if *testVersion == "master" {
+			// the kubernetes binaries should've already been built above
+			// or by the user if --localK8sDir was set, these binaries should be copied to the
+			// path sent to kubetest2 through its --artifacts path
+			klog.Infof("Copying kubernetes binaries to path=%s to run the tests", kubetestDumpDir)
```

Comment:
kubetestdumpDir may be empty, in that case maybe it would be better to set it to ./? (and will it work in this case, or have we implicitly assumed an artifactsDir if we're testing master?)

Reply:
thanks for catching this, I saw that if we don't set `--artifacts` then kubetest2 assumes that it's `./_artifacts`, from its help:

```
Flags:
      --artifacts string         top-level directory to put artifacts under for each kubetest2 run, defaulting to "${ARTIFACTS:-./_artifacts}". If using the ginkgo tester, this must be an absolute path. (default "/usr/local/google/home/mauriciopoppe/go/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/_artifacts")
```

I modified it to be that path if `kubetestdumpDir` is empty